test/test_matmul_fp8_cpu.py

```diff
@@ -0,0 +1,87 @@
+# Owner(s): ["module: mkldnn"]
+
+from typing import Optional
+
+import torch
+from torch.testing._internal.common_device_type import instantiate_device_type_tests
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+# Protects against includes accidentally setting the default dtype
+assert torch.get_default_dtype() is torch.float32
+
+e4m3_type = torch.float8_e4m3fn
+e5m2_type = torch.float8_e5m2
+E4M3_MAX_POS = torch.finfo(torch.float8_e4m3fn).max
+E5M2_MAX_POS = torch.finfo(torch.float8_e5m2).max
+
+
+class TestFP8MatmulCpu(TestCase):
+    def _test_tautological_mm(
+        self,
+        device: str = "cpu",
+        x_dtype: torch.dtype = e4m3_type,
+        y_dtype: torch.dtype = e4m3_type,
+        out_dtype: Optional[torch.dtype] = None,
+        size: int = 16,
+    ) -> None:
+        x_fp8 = torch.rand(size, size, device=device).to(x_dtype)
+        y_fp8 = torch.eye(size, device=device, dtype=y_dtype).t()
+        out_fp32 = torch.mm(x_fp8.to(torch.float), y_fp8.to(torch.float))
+        scale_a = torch.tensor(1.0, device=device)
+        scale_b = torch.tensor(1.0, device=device)
+        out_fp8 = torch._scaled_mm(x_fp8, y_fp8, scale_a, scale_b, out_dtype=out_dtype)
+        if out_dtype is not None:
+            self.assertEqual(out_dtype, out_fp8.dtype)
+        self.assertEqual(out_fp32, out_fp8.to(torch.float))
```

Comment:
This looks like a copy-n-paste of https://github.com/pytorch/pytorch/blob/5c2584a14c2283514703a17cba0a57c8bfb0d977/test/test_matmul_cuda.py#L334
In that case please move the base class either to torch/testing/_internal/ or to some common test file, say `test_matmul_fp8.py` as part of separate PR and than extend it to CPU specific cases

Reply:
I moved `TestFP8Matmul` to test/test_matmul_fp8.py, and add CPU related changes.